MetFamily/ui.R

```diff
@@ -1,1232 +1,1243 @@
-                wellPanel(
-                  h4("Metabolite family selection"),
-                  bsTooltip(id = "familyCount2", title = "The number of available metabolite families which were annotated among the MS\u00B9 features", placement = "bottom", trigger = "hover"),
-                  verbatimTextOutput(outputId = "familyCount2"),
-                  DT::dataTableOutput("familySelectionTable"),
-                  conditionalPanel(
-                    condition = "output.metaboliteFamilySelected",
-                    #condition = "length(input$familySelectionTable_rows_selected) == 1",
-                    h4("Metabolite family properties"),
-                    bsTooltip(id = "featureCountForFamily", title = "The number of MS\u00B9 features of the selected metabolite family", placement = "bottom", trigger = "hover"),
-                    verbatimTextOutput(outputId = "featureCountForFamily"),
-                    fluidRow(
-                      column(width = 4,
-                             bsTooltip(id = "selectMetaboliteFamily", title = "Press to select this metabolite family", placement = "bottom", trigger = "hover"),
-                             actionButton(inputId = "selectMetaboliteFamily", label = "Select", class="btn-success", width = "100%")
-                      ),##column
-                      column(width = 4,
-                             bsTooltip(id = "renameMetaboliteFamily", title = "Press to rename this metabolite family annotation", placement = "bottom", trigger = "hover"),
-                             actionButton(inputId = "renameMetaboliteFamily", label = "Rename", class="btn-success", width = "100%")
-                      ),##column
-                      column(width = 4,
-                             bsTooltip(id = "removeMetaboliteFamily", title = "Press to remove this metabolite family annotation", placement = "bottom", trigger = "hover"),
-                             actionButton(inputId = "removeMetaboliteFamily", label = "Remove", class="btn-danger", width = "100%")
-                      )##column
-                    ),##row
-                    conditionalPanel(
-                      condition = "output.classifierLoaded",
-                      bsTooltip(id = "metaboliteFamilyComparisonClass", title = "Please choose the class for comparison", placement = "bottom", trigger = "hover"),
-                      selectInput(multiple = FALSE, inputId = "metaboliteFamilyComparisonClass", label = "Compare to class", selected = "[init]", choices = c("[init]"), selectize = FALSE)
-                    ),
-                    plotOutput(height = 250, 
-                               outputId = "fragmentPlot2", 
-                               #hover    = "fragmentPlot2_hover",
-                               hover    = hoverOpts(
-                                 id = "fragmentPlot2_hover",
-                                 delay = 50, 
-                                 delayType = "debounce"
-        
```

Comment:
Is this a mix of reformatting and some actual change ? Can't review here.

Reply:
https://github.com/ipb-halle/MetFamily/pull/54/files/3e41e23cdae91eb739e0309f9826253d7b7a99c0#r384733331

Whhopps. I have not noticed the change of the whole file.  Indeed only one line was changed.